public/index.pug

```diff
@@ -315,28 +346,14 @@ html(lang='en-US')
                   alt=`${icon.title} icon`
                 )
             .grid-item__row
-              if icon.guidelines
```

Comment:
Let me know when you have implemented the view in detail and I'll continue the review 👍🏼

Reply:
will do!

Reply:
Icon detail view is now added. 
Set width of 400px; I think this sufficient but there may be other opinions. 
I've restored guidelines, and licenses in this view, so that info that is no longer present in the layouts can still be seen here.
I've also added a color SVG download from this view.